src/metrics/prometheus.go

```diff
@@ -205,3 +211,21 @@ func (m *metrics) pushMetrics() int {
 	log.Debug("Push #%d of metrics in %0.3fs", m.pushes, time.Since(start).Seconds())
 	return 0
 }
+
+// deriveLabelValue runs a command and returns its output.
+// It returns the empty string on error; we assume it's better to keep the set of labels constant on failure.
```

Comment:
I think it makes sense to keep a constant number. 

Alternatively, could we fail hard since this is done once in the `initMetrics`? And if we do, what do you think about checking that the command produced only one value (or one line) and failing otherwise? I don't know how the Prometheus API/UI will handle newlines, and besides, anything that is not 1 value feels like an error to me